source/common/ssl/context_config_impl.cc

```diff
@@ -25,10 +25,13 @@ const std::string ContextConfigImpl::DEFAULT_CIPHER_SUITES =
     "AES256-SHA256:"
     "AES256-SHA";
 
+const std::string ContextConfigImpl::DEFAULT_ECDH_CURVES = "X25519:P-256";
```

Comment:
Curve selection LGTM. Yeah, X25519 is generally the preferred curve, but P-256 is still needed to talk to all the older clients out there.